webpack/assets/javascripts/react_app/redux/actions/toasts/index.js

```diff
@@ -0,0 +1,32 @@
+import {
+  TOASTS_ADD,
+  TOASTS_HIDE,
+  TOASTS_DELETE
+  // TOASTS_STORE_IN_LOCALSTORAGE,
+  // TOASTS_GET_FROM_LOCALSTORAGE
+} from '../../consts';
+import store from '../../../redux';
+
+// provide mechanism to persist toasts to localStorage
+export const addToast = (toast) => {
+  const payload = Object.assign({}, toast, {visible: true});
```

Comment:
This seems more robust to me but I'll remove it if you like. I'm sort of 55/45 on the issue.